test/Functional/cypress-commands/src/tenants.js

```diff
@@ -37,14 +37,27 @@ Cypress.Commands.add('newTenant', function(tenantInfo) {
   cy.siteSetup(tenantInfo);
 });
 
-Cypress.Commands.add("createTenant", ({ name, prefix, setupRecipe, description }) => {
+Cypress.Commands.add("createTenant", ({ name, prefix, setupRecipe, description}) => {
   // We create tenants on the SaaS tenant
   cy.visit("/Admin/Tenants");
   cy.btnCreateClick();
   cy.get("#Name").type(name, {force:true});
   cy.get("#Description").type(`Recipe: ${setupRecipe}. ${description || ''}`, {force:true});
   cy.get("#RequestUrlPrefix").type(prefix, {force:true});
   cy.get("#RecipeName").select(setupRecipe);
-  cy.get("#DatabaseProvider").select('Sqlite')
+  cy.get("body").then($body => {
+    const db = $body.find("#DatabaseProvider");
+    // if a database provider is already specified by an environment variable.. leave it as is
+    // this assumes that if you set the provider, you also set the connectionString
+    if (db.length > 0 && db.val() == "") {
+      db.val('Sqlite');
+    } else {
+        //set the tablePrefix to the name.
+        const prefix = $body.find("#TablePrefix");
+        if(prefix.length > 0){
+            prefix.val(name);
```

Comment:
Here you are using `name` for the table prefix, rather than `prefix`- I know they're calculated the same, so call me Mr Picky ;)

Reply:
i needed to keep the prefix really small because I hit an issue with postgresql table size. Some of our indexes are long ;)

Reply:
Yes, I ran into the same problem :) Same with MySql

What I meant was you have a prefix variable, and a name variable, (which are calculated to the same value), but instead of using the prefix variable here you use the name. 

Ah but I suddenly got it. The prefix variable is the url prefix, not the table prefix.

Too many prefixes... confused me

Reply:
Ahhhh I get it. The only reason it's there was to allow flexibility for folks that want a different prefix. I guess it is a bit redundent. prefix should be renamed to urlPrefix to avoid confusion. Maybe also introduce a tablePrefix variable ? :P